src/test/index.js

```diff
@@ -0,0 +1,37 @@
+import chai from 'chai';
```

Comment:
create a testHelpers folder in the test folder and configure chai there.  this will help eradicate line 1 to 9 from this file.

Reply:
yes nice, I will do that

Reply:
done